relay-general/src/protocol/clientsdk.rs

```diff
@@ -54,6 +54,13 @@ pub struct ClientSdkInfo {
     #[metastructure(skip_serialization = "empty_deep")]
     pub packages: Annotated<Array<ClientSdkPackage>>,
 
+    /// The installation mechanism for the SDK. _Optional._
+    ///
+    /// This can be a package manager like `npm` or `cocoapods` or a custom installation mechanism
+    /// like `cdn`.
+    #[metastructure(skip_serialization = "empty")]
+    pub source: Annotated<String>,
```

Comment:
Is this field guaranteed to be free of sensitive data? That is, will it have an enum-like cardinality, or could this also contain paths or URLs with usernames, etc.

Reply:
Yes - it will have enum-like cardinality. The values will be fixed per SDK, and free to all possible PII.

I can update the docstring to include these details.